app/models/concerns/tiny_mce_images.rb

```diff
@@ -73,7 +73,10 @@ def clone_tinymce_assets(target, team)
 
         tiny_img_clone.transaction do
           tiny_img_clone.save!
-          tiny_img_clone.image.attach(io: tiny_img.image.open, filename: tiny_img.image.filename.to_s)
+          tiny_img_clone.image.attach(io: StringIO.new(tiny_img.image.download),
```

Comment:
I would use temp files instead of loading file content to memory as file size is limited by FILE_MAX_SIZE_MB and it can be quite big